Mathlib/Algebra/Lie/Submodule.lean

```diff
@@ -1447,11 +1447,14 @@ def LieIdeal.topEquiv : (⊤ : LieIdeal R L) ≃ₗ⁅R⁆ L :=
   LieSubalgebra.topEquiv
 #align lie_ideal.top_equiv LieIdeal.topEquiv
 
+-- These lemmas have always been bad (#7657), but leanprover/lean4#2644 made `simp` start noticing
 @[simp]
```

Comment:
```suggestion
-- These lemmas have always been bad (#7657), but leanprover/lean4#2644 made `simp` start noticing
@[simp]
```
```suggestion
-- These lemmas have always been bad (#7657), but leanprover/lean4#2644 made `simp` start noticing
@[simp]
```
```suggestion
-- These lemmas have always been bad, but leanprover/lean4#2644 made `simp` start noticing
@[simp, nolint simpNF]
```
#7657 can't be relevant here, this isn't `simps`! Are we sure this `nolint` is even needed?

Reply:
(same applies to the rest of the file)